en/_posts/2015-02-14-how-to-run-bottle-uwsgi-nginx.markdown

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title: "Running your Python Bottle application on Nginx using uWSGI"
+categories: development
+tags: guide, getting-started, python
+lang: en
+author: thoorium
+---
+
+[Bottle](http://bottlepy.org/) is a fast, simple and lightweight WSGI micro web-framework for [Python](https://www.python.org/). Coupled with [uWSGI](https://uwsgi-docs.readthedocs.org/en/latest/) and [Nginx](http://nginx.org/) bottle will reach high performances to serve and distribute your content.
```

Comment:
Add short intro on what is uWSGI, and why it is so important? Maybe highlight the differences with the intuitive approach of using NGinx as plain HTTP reverse proxy.